subprojects/execution/src/main/java/org/gradle/internal/execution/UnitOfWork.java

```diff
@@ -109,23 +111,25 @@ default void visitImplementations(ImplementationVisitor visitor) {
     }
 
     /**
-     * Visit all inputs of the work.
+     * Visit regular inputs of the work.
      */
-    void visitInputs(InputVisitor visitor);
+    default void visitIdentityInputs(InputVisitor visitor) {}
+
+    /**
+     * Visit regular inputs of the work.
```

Comment:
Can you clarify what "regular" inputs and "identity" inputs mean?

Reply:
Updated the Javadoc.